doc/instruction_decode_execute.rst

```diff
@@ -64,10 +64,31 @@ Other blocks use the ALU for the following tasks:
 * It computes memory addresses for loads and stores with a Reg + Imm calculation
 * The LSU uses it to increment addresses when performing two accesses to handle an unaligned access
 
-Support for the RISC-V Bitmanipulation Extension (Document Version 0.92, November 8, 2019) is enabled via the parameter ``RV32B``.
-This feature is *EXPERIMENTAL* and the details of its impact are not yet documented here.
-Currently the Zbb, Zbs, Zbp, Zbe, Zbf, Zbc, Zbr and Zbt sub-extensions are implemented.
-The rotate instructions `ror` and `rol` (Zbb), ternary instructions `cmov`, `cmix`, `fsl` and `fsr` as well as cyclic redundancy checks `crc32[c]` (Zbr) are completed in 2 cycles. All remaining instructions complete in one cycle.
+Bit Manipulation Extension
+  Support for the RISC-V Bitmanipulation Extension (Document Version 0.92, November 8, 2019) is enabled via the enumerated parameter ``RV32B`` defined in :file:`rtl/ibex_pkg.sv`. This feature is *Experimenal*.
+  There are two versions of the bit manipulaition extension available: The balanced implementation comprises a set of sub-extensions aiming for good benefits at a readonable area overhead. The full implementation comprises all 32 bit instructions defined in the extension. The following table lists the implemented instructions in each version. Multi-cycle instructions are completed in 2 cycles. All remaining instructions complete in a single cycle.
+
+  +---------------------------+---------------+--------------------------+
+  | Z-Extension               | Version       | Multi-Cycle Instructions |
+  +===========================+===============+==========================+
+  | Zbb (Base)                | Balanced/Full | rol, ror[i]              |
+  +---------------------------+---------------+--------------------------+
+  | Zbs (Single-bit)          | Balanced/Full | None                     |
+  +---------------------------+---------------+--------------------------+
+  | Zbp (Permutation)         | Full          | None                     |
+  +---------------------------+---------------+--------------------------+
+  | Zbp (Bit extract/deposit) | Full          | All                      |
+  +---------------------------+---------------+--------------------------+
+  | Zbf (Bit-field place)     | Balanced/Full | All                      |
+  +---------------------------+---------------+--------------------------+
+  | Zbc (Carry-less mltiply)  | Full          | None                     |
+  +---------------------------+---------------+--------------------------+
+  | Zbr (Crc))                | Full          | All                      |
+  +---------------------------+---------------+--------------------------+
+  | Zbt (Ternary))            | Balanced/Full | All                      |
+  +---------------------------+---------------+--------------------------+
```

Comment:
It would be nice if you could also make a comment on the situation regarding `sext.b/sext.h`. These were not classified in 0.92 but you implemented them as part of the balanced configuration.

Reply:
Done.